explore-analyze/manage-access-to-ai-assistant.md

```diff
@@ -37,10 +35,16 @@ To manage these settings, go to the **GenAI Settings** page by using the navigat
 
 The **GenAI Settings** page has the following settings:
 
-- **Default AI Connector**: Use this setting to specify which connector is selected by default. This affects all AI-powered features, not just AI Assistant. 
-- **Disallow all other connectors**: Enable this setting to prevent connectors other than the default connector specified above from being used in your space. This affects all AI-powered features, not just AI Assistant. 
-- **AI feature visibility**: This button opens the current Space's settings page. Here you can specify which features should appear in your environment, including AI-powered features. 
-- **AI Assistant visibility**: This setting allows you to choose which AI Assistants are available to use and where. You can choose to only show the AI Assistants in their native solutions, in other {{kib}} pages (for example, Discover, Dashboards, and Stack Management), or select **Hide all assistants** to disable AI Assistant throughout {{kib}}.
+- **Default AI Connector**: Use this setting to specify which connector is selected by default when you access AI-powered features. Default setting: **No default connector**.
+  - If **No default connector** is selected, AI-powered features will default to the connector that was most recently used in your environment. 
+  - If **Elastic Managed LLM** or a custom LLM connector is selected, AI-powered features will default to that connector regardless of which connector was used most recently in your environment. 
+- **Disallow all other connectors**: When this setting is disabled, whenever you use an AI-powered feature you can select which connector should power it. Enable it to prevent connectors other than the default connector specified above from being used in your space. Default setting: disabled.
```

Comment:
```suggestion
- **Disallow all other connectors**: When this setting is disabled, whenever you use an AI-powered feature you can select which connector should power it. Enable it to prevent connectors other than the default connector from being used in your space. Default setting: disabled.
```